examples/nlp/token_classification/punctuation_capitalization.py

```diff
@@ -110,14 +128,21 @@
     help="Frequency of saving checkpoint \
                     '-1' - step checkpoint won't be saved",
 )
-parser.add_argument("--loss_step_freq", default=250, type=int, help="Frequency of printing loss")
+parser.add_argument(
+    "--eval_epoch_freq", default=1, type=int, help="Frequency of evaluation",
+)
+parser.add_argument(
+    "--loss_log_freq", default=50, type=int, help="Frequency of logging loss values, '-1' - at the end of the epoch",
+)
 parser.add_argument(
     "--use_weighted_loss_punct",
     action='store_true',
     help="Flag to indicate whether to use weighted loss \
                     to mitigate classs unbalancing for the punctuation task",
 )
-
+parser.add_argument("--project", default=None, type=str, help='Project name for tracking with Weights and Biases')
```

Comment:
How about project_name or something to show it is for wandb? like wandb_project and wandb_exp? or wb_project_name and wb_exp_name?

Reply:
done